samples/drivers/peci/src/main.c

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <drivers/peci.h>
+#include <soc.h>
+
+#define TASK_STACK_SIZE 1024
```

Comment:
Make any required macro unsigned if needed

Reply:
TASK_STACK_SIZE is for size_t, but made other constants unsigned